pkg/verifier/cosign/cosign.go

```diff
@@ -594,3 +623,27 @@ func processAKVSignature(sigEncoded string, staticSig oci.Signature, publicKey c
 	}
 	return hashType, staticSig, nil
 }
+
+// verificationMessage returns a string list of all verifications performed
+// based on https://github.com/sigstore/cosign/blob/5ae2e31c30ee87e035cc57ebbbe2ecf3b6549ff5/cmd/cosign/cli/verify/verify.go#L318
+func verificationMessage(bundleVerified bool, co *cosign.CheckOpts) []string {
```

Comment:
I feel the function name is not clear enough, but to be honest, I don't have a better name.